tailwind.config.ts

```diff
@@ -15,11 +15,12 @@ export default {
         "black-800": "#141414",
       },
       colors: {
-        "league-gray-800": "#383838",
-        "league-gray-700": "#B8B8B8",
-        "league-gray-300": "#DADADA",
-        "league-gray-200": "#DBDBDB",
-        "league-gray-100": "#E8E7E7",
+        "league-gray-1": "#ECECEC",
```

Comment:
preco takto? v tailwinde je zauzivana ta syntax so stovkami

Reply:
U tailwind defaultnich barev, ale tohle jsou custom, tak je na nás, jak si je pojmenujeme. Takto si méně spleteš gray-200 s league-gray-200.

Reply:
no ze si to ty johanecko